dbt_platform_helper/utils/platform_config.py

```diff
@@ -1,17 +1,32 @@
-from pathlib import Path
-
 import yaml
 
+from pathlib import Path
+
 from dbt_platform_helper.constants import PLATFORM_CONFIG_FILE
-from dbt_platform_helper.utils.validation import load_and_validate_platform_config
+
+
+def read_config_file():
+    if Path(PLATFORM_CONFIG_FILE).exists():
+        return Path(PLATFORM_CONFIG_FILE).read_text()
+
+
+def load_config_file():
```

Comment:
Probably worth naming this to reflect that it ignores errors by design. To stop people adding validation in the future or using it where they should be using the load_and_validate_platform_config function.